audio-worklet/src/bitcrusher-script-processor.js

```diff
@@ -49,8 +49,13 @@ class Bitcrusher {
     this.input.connect(this.node_).connect(this.output);
 
     // Index and previousSample defined as globals to handle block transitions.
-    this.index_ = 0;
-    this.previousSample_ = 0;
+    // There is one of each for every channel to handle multi-channel input.
+    this.indexes_ = new Array();
+    this.previousSamples_ = new Array();
+    for (let i = 0; i < channels; i++) {
+      this.indexes_.push(0);
+      this.previousSamples_.push(0);
+    }
```

Comment:
If you create the arrays with the right size, you can then easily do
```
this.indexes_.fill(0);
this.previousSamples_.fill(0)
```
This is a lot more obvious than using a loop.